src/test_servers.py

```diff
@@ -1 +1,24 @@
+# -*- coding: utf-8 -*-
 """Tests for http-server client and server modules."""
+
+import pytest
+
+
+MSG_TABLE = [
+    ('Msg'),
+    ('MsgEIGHT'),
+    ('Msg long is a really long buffer'),
+    ('MsgEIGHTMsgEIGHTMsgEIGHT'),
+
+]
+
+
```

Comment:
Good job parametrizing your test!  The only critique I have, is that now your docstrings are lying, and the test function name is no longer accurate.